### DISCUSSION TITLE
希望可以登录微软商店，方便XBOX用户

### DISCUSSION THREAD

**Discussion by zjpChina** [ywmoyue/biliuwp-lite#134] [General]
如题，希望这款Bilibili第三方客户端可以登录微软商店，目前XBOX上的Bilibili要不就是官方做的太落后，要不就是第三方接口失效没有大佬更新，希望这一款软件可以登录微软商店！

**Comment by ywmoyue** [comment]
1.我没有xbox，没做过xbox应用
2.我简单对比了下biliuwp-lite和哔哩的代码，发现biliuwp-lite没有哔哩那样的跨平台相关的设计
3.建议可以看看《22与33的奇妙冒险》和 [《哔哩》](https://github.com/Richasy/Bili.Uwp/) 这两款支持xbox的应用，哔哩还是有人提PR的

**Reply by shichaoufo** [reply]
逍遥橙子的另一个应用聚合直播也没有发布到xbox上面，只有windows商店版，但是有另一个开发者听了用户的建议直接拿逍遥橙子的源代码重新编译就发布到xbox上了，不知道大佬你这个应用是不是也可以简单的直接发布到xbox上
![edcb6667b1a1f44ea06bcb74ac1f8d9](https://github.com/ywmoyue/biliuwp-lite/assets/18585420/5edb47f7-1316-4379-8baf-1f1427edee09)
![504c88329f678ceac69c2e63dc48f40](https://github.com/ywmoyue/biliuwp-lite/assets/18585420/1c24af59-9a9e-459f-9851-3068a7b9b2f9)
